Gemfile

```diff
@@ -12,6 +12,7 @@ gem 'bootsnap', require: false # Reduces boot times through caching; required in
 gem 'feedjira' # Parse RSS feeds
 gem 'good_job' # Multithreaded, Postgres-based, ActiveJob backend for Ruby on Rails
 gem 'image_processing' # Use Active Storage variants
+gem 'inline_svg' # Render inline SVGs
```

Comment:
Didn't know about this gem - I've written a similar thing in various projects to fetch assets from a specific folder